src/ethereum_test/base_test.py

```diff
@@ -59,16 +57,15 @@ def verify_post_alloc(post: Mapping[str, Account], alloc: Mapping[str, Any]):
     """
     for address, account in post.items():
         address = normalize_address(address)
-        if account is None:
-            # If an account is None in post, it must not exist in the
-            # alloc.
-            if address in alloc:
-                raise Exception(f"found unexpected account: {address}")
-        else:
-            if address in alloc:
-                account.check_alloc(address, alloc[address])
+        if account is not None:
+            if account == Account.NONEXISTENT:
+                if address in alloc:
```

Comment:
Minor nit: I think `post` and `alloc` could be more descriptive. I'm not actually sure which value is the expected and which was computed. I _think_ `post` is expected from test and `alloc` is the computed value? In geth we often use `got`/`want` or `got`/`exp`.

Reply:
Fixed!